test/parallel/test-fs-read-stream-inherit.js

```diff
@@ -52,7 +52,7 @@ file.on('close', function() {
   //assert.equal(fs.readFileSync(fn), fileContent);
 });
 
-var file3 = fs.createReadStream(fn, Object.create({encoding: 'utf8'}));
+var file3 = fs.createReadStream(fn, {encoding: 'utf8'});
```

Comment:
@jasnell If that breakage is not acceptable, we can have a function which is similar to `_extend`, but with `for..in`, instead of `Object.keys`.